docs/guides/latest/advance-use-case/instant-backup.md

```diff
@@ -0,0 +1,378 @@
+
+```yaml
+$ kubectl get pod -n demo stash-demo-9bff9fd4f-xvt77 -o yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  labels:
+    app: stash-demo
+    pod-template-hash: 9bff9fd4f
+  name: stash-demo-9bff9fd4f-xvt77
+  namespace: demo
+  uid: 044cfa14-9b2f-47e8-ac15-399b9addd61d
+  ...
+spec:
+  containers:
+  - args:
+    - echo sample_data > /source/data/data.txt && sleep 3000
+    command:
+    - /bin/sh
+    - -c
+    image: busybox
+    imagePullPolicy: IfNotPresent
+    name: busybox
+    volumeMounts:
+    - mountPath: /source/data
+      name: source-data
+    - mountPath: /var/run/secrets/kubernetes.io/serviceaccount
+      name: default-token-dp4pv
+      readOnly: true
+  - args:
+    - run-backup
+    - --backup-configuration=deployment-backup
+    - --secret-dir=/etc/stash/repository/secret
+    - --enable-cache=true
+    - --max-connections=0
+    - --metrics-enabled=true
+    - --pushgateway-url=http://stash-operator.kube-system.svc:56789
+    - --enable-status-subresource=true
+    - --use-kubeapiserver-fqdn-for-aks=true
+    - --enable-analytics=true
+    - --logtostderr=true
+    - --alsologtostderr=false
+    - --v=3
+    - --stderrthreshold=0
+    env:
+    - name: NODE_NAME
+      valueFrom:
+        fieldRef:
+          apiVersion: v1
+          fieldPath: spec.nodeName
+    - name: POD_NAME
+      valueFrom:
+        fieldRef:
+          apiVersion: v1
+          fieldPath: metadata.name
+    image: suaas21/stash:restore-pvc-tpl_linux_amd64
+    imagePullPolicy: IfNotPresent
+    name: stash
+    volumeMounts:
+    - mountPath: /etc/stash
+      name: stash-podinfo
+    - mountPath: /etc/stash/repository/secret
+      name: stash-secret-volume
+    - mountPath: /tmp
+      name: tmp-dir
+    - mountPath: /source/data
+      name: source-data
+    - mountPath: /var/run/secrets/kubernetes.io/serviceaccount
+      name: default-token-dp4pv
+      readOnly: true
+  volumes:
+  - name: source-data
+    persistentVolumeClaim:
+      claimName: source-data
+  - emptyDir: {}
+    name: tmp-dir
+  - downwardAPI:
+      defaultMode: 420
+      items:
+      - fieldRef:
+          apiVersion: v1
+          fieldPath: metadata.labels
+        path: labels
+    name: stash-podinfo
+  - name: stash-secret-volume
+    secret:
+      defaultMode: 420
+      secretName: gcs-secret
+  - name: default-token-dp4pv
+    secret:
+      defaultMode: 420
+      secretName: default-token-dp4pv
+  ...
+...
+```
+
+**Verify CronJob:**
+
+It will also create a `CronJob` with the schedule specified in `spec.schedule` field of `BackupConfiguration` crd.
+
+Verify that the `CronJob` has been created using the following command,
+
+```console
+$ kubectl get backupconfiguration -n demo 
+NAME                TASK   SCHEDULE       PAUSED   AGE
+deployment-backup          */40 * * * *            6m41s
+```
+
+## Backup Instant
+
+Now, we will take instant backup of the data volumes. for that, we will deploy a `BackupSession` object to trigger instant backup.
```

Comment:
Now, we will trigger an instant backup of Deployment's volumes that we have configured in the previous section. In order to do that, we have to create a `BackupSession` object pointing to the respective `BackupConfiguration` object.